apps/wallet/src/ui/app/staking/delegation-detail/DelegationDetailCard.tsx

```diff
@@ -108,10 +116,9 @@ export function DelegationDetailCard({ validatorAddress, stakedId }: DelegationD
         staked: stakedId,
     }).toString()}`;
 
-    // check if the validator is in the committee members list
-    const hasInactiveValidatorDelegation = !system?.committeeMembers?.find(
-        ({ stakingPoolId }) => stakingPoolId === validatorData?.stakingPoolId,
-    );
+    const isValidatorCommitteeMember = isCommitteeMember(validatorAddress);
+    const isAnActiveValidator = isActiveValidator(validatorAddress);
```

Comment:
nit, so it follows the same style as above
```suggestion
    const isValidatorActive = isActiveValidator(validatorAddress);
```